src/main/scala/com/linkedin/feathr/swj/transformer/FeatureTransformer.scala

```diff
@@ -93,6 +93,7 @@ object FeatureTransformer {
     // The order of the feature columns should follow the order of the specified sliding window
     // features. In addition, the feature columns should be in front of join key and timestamp
     // columns. These will guarantee we can fetch corresponding feature columns using index.
+    println(s"SQL: ${sql}")
```

Comment:
should we remove this print?

Reply:
Let's keep it as-is, this is helpful for diagnostic, especially when the issue happens on the customer site.
Also, it's not a security leak, no info more sensitive than the feature definitions will be exposed in the output.

Reply:
Talked offline, shall we call it something like "Generated SQL statement" so that it's clear for debug purpose?

Reply:
Fixed